features/step_definitions/materials_collections_steps.rb

```diff
@@ -39,28 +39,32 @@
   end
 end
 
-When /^I drag the (\d+)(?:st|nd|rd|th) material in the materials collection "([^"]*)" to the (top|bottom)/ do |start_position, name, end_position|
+def set_sortable_sequence(sortable_container_id, sequence)
+  page.execute_script("Sortable.setSequence(jQuery('##{sortable_container_id}')[0], #{sequence})")
```

Comment:
Does it not work to simulate a drag with chromedriver?  
This shouldn't hold up the PR, I'm just curious.

Reply:
No, drag n' drop works more or less as it did before. However, this feature spec was always broken (see the FIXME note removed in the diff). In chrome the behavior was slightly different in that elements drag n' dropped on top of other elements in the sortable list would end up above the target element, not below. In Firefox it was the opposite.

What I've done here adds complexity because it requires knowledge of Sortable internals. However, it is the only way I could think of to resolve the FIXME for good.

@knowuh was recently working on these specs in another branch. Perhaps he can chime in on this too.